sklearn/model_selection/_validation.py

```diff
@@ -476,7 +477,9 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
                     train_scores = error_score
             warnings.warn("Classifier fit failed. The score on this train-test"
                           " partition for these parameters will be set to %f. "
-                          "Details: \n%r" % (error_score, e), FitFailedWarning)
+                          "Details: \n%s" %
```

Comment:
the classifier here should be estimator, right?

Reply:
Yup, I am still fixing it, I should change the name of this thread to WIP. Thanks for pointing it out anyways.